docs/root/version_history/current.rst

```diff
@@ -57,6 +57,8 @@ New Features
 * ext_authz filter: added support for emitting dynamic metadata for both :ref:`HTTP <config_http_filters_ext_authz_dynamic_metadata>` and :ref:`network <config_network_filters_ext_authz_dynamic_metadata>` filters.
 * grpc-json: support specifying `response_body` field in for `google.api.HttpBody` message.
 * hds: added :ref:`cluster_endpoints_health <envoy_v3_api_field_service.health.v3.EndpointHealthResponse.cluster_endpoints_health>` to HDS responses, keeping endpoints in the same groupings as they were configured in the HDS specifier by cluster and locality instead of as a flat list.
+* hot restart: added :option:`--socket-path` to configure UDS path in filesystem.
+* hot restart: added :option:`--socket-mode` to set permission on :option:`--socket-path`.
```

Comment:
Only need a single line/entry for the feature; maybe add a `:ref:` to cli.rst

Reply:
` * hot restart: added :option:`--socket-path` and :option:`--socket-mode` to configure UDS path in the filesystem and set permission to it.`

Make sense?

Reply:
Yes, that sounds good.